src/platform/Zephyr/BLEManagerImpl.h

```diff
@@ -111,6 +112,7 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
     bool mSubscribedConns[CONFIG_BT_MAX_CONN];
     bt_gatt_indicate_params mIndicateParams[CONFIG_BT_MAX_CONN];
     bt_conn_cb mConnCallbacks;
+    bool mIsFastAdvertisingOn;
```

Comment:
I can see there's `kFastAdvertisingEnabled` flag. Could it be used instead a new variable?

Reply:
Done